public/app/plugins/datasource/loki/components/monaco-query-field/MonacoQueryField.tsx

```diff
@@ -203,7 +212,11 @@ const MonacoQueryField = ({ history, onBlur, onRunQuery, initialValue, datasourc
             onTypeDebounced(query);
             monaco.editor.setModelMarkers(model, 'owner', markers);
           });
-          const dataProvider = new CompletionDataProvider(langProviderRef.current, historyRef);
+          const dataProvider = new CompletionDataProvider(
+            langProviderRef.current,
+            historyRef,
+            timeRange ?? langProviderRef.current.getDefaultTimeRange()
```

Comment:
Why do we need to ask the `language provider` for the default timerange? I would use the function from grafana data, or change the language provider to tell me its current time range, but it shouldn't know which one is the _default_.

Reply:
Good point, we don't need to default to anything as `timeRange` is alwyas optional prop and if not filled, the default time range will be used. Updated in https://github.com/grafana/grafana/pull/78450/commits/f6b762a95452fd5601684c854bd41261e6031ddd